Code/Mantid/Framework/DataHandling/src/LoadSpiceXML2DDet.cpp

```diff
@@ -437,5 +518,150 @@ MatrixWorkspace_sptr LoadSpiceXML2DDet::createMatrixWorkspace(
   return outws;
 }
 
+//----------------------------------------------------------------------------------------------
+/** Set up sample logs from table workspace loaded where SPICE data file is
+ * loaded
+ * @brief LoadSpiceXML2DDet::setupSampleLogFromSpiceTable
+ * @param matrixws
+ * @param spicetablews
+ * @param ptnumber
+ */
```

Comment:
No, doxygen won't complain.  And these parameter descriptions were added to the code automatically by Qt Creator.  In the long run, I should add detailed description.  Thanks for pointing out this.